app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,17 @@
+class CommentsController < ApplicationController
+  def show
+    @tweet = Tweet.find_by(id: params[:id])
+    @comments = Comment.where(tweet_id: @tweet.id)
+  end
+  def new
+  end
```

Comment:
メソッド間は改行しましょう、見づらいです。
```suggestion
  def show
    @tweet = Tweet.find_by(id: params[:id])
    @comments = Comment.where(tweet_id: @tweet.id)
  end
 
  def new
  end
```